misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/expression/MethodCallExpressionEvaluator.java

```diff
@@ -164,7 +165,7 @@ private Value invokeObjectMethod(BVariable resultVar) throws EvaluationException
                 throw createEvaluationException(CLASS_NOT_FOUND, className);
             }
 
-            Optional<MethodSymbol> objectMethodDef = findObjectMethodInClass(classDef.get(), methodName);
+            Optional<MethodSymbol> objectMethodDef = findObjectMethodInClass(classDef.get(), methodName, null);
```

Comment:
Shall we have an overloaded method definition for `findObjectMethodInClass` so that we won't have to pass `null` in here?